tests/www/views/test_views_log.py

```diff
@@ -163,6 +169,22 @@ def tis(dags, session):
     clear_db_runs()
 
 
+@pytest.fixture
+def create_expected_log_file(log_path, tis):
+    ti, _ = tis
+    handler = FileTaskHandler(log_path)
+
+    def create_expected_log_file(try_number):
+        ti.try_number = try_number - 1
+        handler.set_context(ti)
+        handler.emit(logging.makeLogRecord({"msg": "Log for testing."}))
+        handler.flush()
+
+    yield create_expected_log_file
+    # delete logs to minimize tests interference
+    shutil.rmtree(log_path)
```

Comment:
Not needed, pytest cleans things in the temp directory automatically.

Reply:
@uranusjr pytest cleans directories after exiting defined scope. `log_path` fixture requires `module` scope to be used by `log_app` fixture (which also has `module` scope). But `log_path` should be cleared after each test, because almost all tests reference to the same Dag.

Reply:
That makes sense, thanks. Would it be a good idea to only delete the generated log file instead though? (And add a comment explaining why)

Reply:
@uranusjr in the PR I tried not to make any assumptions regarding log file name. I will add a comment describing why manual log file deletion is required and will find more clear to delete created log file.